### PR TITLE
Add API docs to README.md

### PR DESCRIPTION
**Pull request by zuiderkwast**
Functions and options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,235 @@ Features:
...
+the callback function will executing in the redis client process and
+should not hang or perform any lengthy task.
+
+### `get_clients/1`
+
+```Erlang
+get_clients(server_ref()) -> [client_ref()].
+```
+
+Get all Redis master node clients
```
Comment (bjosv) [inline README.md:124]:
```suggestion
Get all Redis master node clients.
```

File: README.md [hunk 2]
```diff
@@ -14,6 +14,235 @@ Features:
...
+
+Options
+-------
+
+The following options can be passed to `start_link/2`:
+
+* `{try_again_delay, non_neg_integer()}`
+
+  If there is a TRYAGAIN response from Redis then wait
+  this many milliseconds before re-sending the command
```
Comment (bjosv) [inline README.md:142]:
```suggestion
  this many milliseconds before re-sending the command.
```

Comment (bjosv) [reply README.md:142]: Some more missing under `Connection options`

**Review by bjosv — APPROVED** [review]
Nice! Just some nits

File: README.md [hunk 3]
```diff
@@ -14,6 +14,235 @@ Features:
...
+
+* `{redirect_attempts, non_neg_integer()}`
+
+  Only do these many retries or re-sends before giving
+  up and returning the result. This affects ASK, MOVED
+  and TRYAGAIN responses
```
Comment (zuiderkwast) [inline README.md:148]:
```suggestion
  and TRYAGAIN responses.
```

**Pull request merged**